README.md

```diff
@@ -284,6 +284,14 @@ In order to provide full interoperability with NAIF, hifitime uses the NAIF algo
 
 # Changelog
 
+## 3.8.0
```

Comment:
This is a work in progress, but I'm adding this to not forget the release notes.